svg-min/medium.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><rect x="0" fill="none" width="24" height="24"/><g><g><path d="M20.9 7.2l-5.4-2.7h-.2c-.2 0-.4.1-.5.3l-3.1 5.1 3.9 6.4 5.3-9.1c.1 0 .1 0 0 0zM9.9 8.6v5.8l5.1 2.5M15.8 17.3l4.2 2.1c.6.3 1 .1 1-.4V8.8l-5.2 8.5zM9 7.2L3.7 4.5c-.1 0-.2-.1-.3-.1-.2.1-.4.2-.4.5v11.4c0 .3.2.7.5.8l4.7 2.3c.1.1.2.1.3.1.3.1.5-.1.5-.5V7.2z"/></g></g></svg>
```

Comment:
> Oh, I guess it's the optimizer adding this? Strange...

This was intentionally added to Gridicons and therefore inherited to Social Logos when we forked. The purpose of the extra `g` tag is to be able to specify it for CSS (i.e. `.classname svg g`). This is necessary in the case of subpixel `transform` operations without interfering with any `transform`s you might want to add as part of animations or something.